aspnet/mvc/views/razor.rst

```diff
@@ -1,8 +1,729 @@
-.. include:: /../common/stub-topic.txt
+Razor Syntax Reference
+===========================
+`Taylor Mullen <https://twitter.com/ntaylormullen>`__, and `Rick Anderson`_
 
-|stub-icon| Razor Syntax
-========================
+.. contents:: Sections
+  :local:
+  :depth: 1
 
-.. include:: /../common/stub-notice.txt
+What is Razor?
+--------------
+Razor is a markup syntax for embedding server based code into web pages. The Razor syntax consists of Razor markup, C# and HTML. Files containing Razor generally have a *.cshtml* file extension.
 
-.. _issue: https://github.com/aspnet/Docs/issues/130
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+Rendering HTML
+--------------
+
+The default Razor language is HTML. Rendering HTML from Razor is no different than in an HTML file. A Razor file with the following markup:
+
+.. code-block:: none
+
+  <p>Hello World</p>
+
+Is rendered unchanged as ``<p>Hello World</p>`` by the server.
+
+Rendering server code
+----------------------
+
+Razor supports C# and uses the ``@`` symbol to transition from HTML to C#.
```

Comment:
> Razor supports C# and uses the ``@`` symbol to transition from HTML to C#. 

Razor supports C# and uses the ``@`` symbol to transition from HTML to C#. Razor evaluates C# expressions and renders them in the HTML output.